data/isdv4-10g.tablet

```diff
@@ -0,0 +1,15 @@
+# this is for the Wacom pen + touchscreen as found in the Fujitsu T935
+
+[Device]
+Name=Wacom ISDv4 101
+ModelName=
+DeviceMatch=usb:056a:5019
+Class=ISDV4
+Width=11
+Height=6
+IntegratedIn=Display;System
+
+[Features]
+Stylus=true
+Touch=true
+Buttons=1
```

Comment:
`Buttons` refers to the number of buttons on the **tablet**, not on the stylus - is this correct? Quick google image search for the T935 doesn't show any buttons.